pkg/kubelet/volume/gce_pd/mount_util.go

```diff
@@ -35,19 +39,36 @@ func getMountRefCount(mounter mount.Interface, mountPath string) (string, int, e
 
 	// Find the device name.
 	deviceName := ""
-	for i := range mps {
-		if mps[i].Path == mountPath {
-			deviceName = mps[i].Device
+	for _, mp := range mps {
```

Comment:
to pick a nit - this is strictly a pessimization, since Go does not have any concept of references.  You're going to end up making a copy of half of the mount table, which actually can be fairly large.